cmd/podman/events.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"github.com/containers/libpod/cmd/podman/cliconfig"
+	"github.com/containers/libpod/pkg/adapter"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	eventsCommand     cliconfig.EventValues
+	eventsDescription = "Monitor podman events"
+	_eventsCommand    = &cobra.Command{
+		Use:   "events [flags]",
+		Short: "show podman events",
+		Long:  eventsDescription,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			eventsCommand.InputArgs = args
+			eventsCommand.GlobalFlags = MainGlobalOpts
+			return eventsCmd(&eventsCommand)
+		},
+		Example: `podman events
+  podman events -f`,
```

Comment:
would be nice to have a little more detailed event here.  I'd keep the first then add the one from your manpage with the filter and what ever you work up for the format.   As a rule of thumb I've been trying to make it 3 examples max.